charts/kube-prometheus-stack/values.yaml

```diff
@@ -580,7 +580,15 @@ grafana:
   ##
   defaultDashboardsEnabled: true
 
-  adminPassword: prom-operator
+  ## Set the default admin username
+  #
+  adminUser: admin
+
+  ## Set the default admin password
+  ## An empty string generates a psuedo-random password
+  ## Otherwise, this value is used
+  #
+  adminPassword: ""
```

Comment:
imho thats a breaking change, even if nobody should use a static password here.
not sure if it should be a bugfix or major breaking change?

please also raise chart version.